src/UnlitBatchShader.ts

```diff
@@ -142,17 +149,38 @@ uniform sampler2DArray map;
 
 in vec2 vUv;
 in vec4 vColor;
-flat in uint vMapIdx;
 flat in vec4 vUVTransform;
+flat in vec4 vMapSettings;
 
 out vec4 outColor;
 
+float applyWrapping(float value, int mode) {
+  if (mode == 0) {
+    // CLAMP_TO_EDGE - default
+    return clamp(value, 0.0, 1.0);
+  } else if (mode == 1) {
+    // REPEAT
+    return fract(value);
+  } else {
+    // MIRRORED_REPEAT
+    float n = mod(value, 2.0);
+    return mix(n, 2.0 - n, step(1.0, n));
+  }
```

Comment:
Yeah, I just excluded those since the glTF spec doesn't have them and we control how everything else is wrapped. Maybe I should log a warning if the wrap mode is unsupported.

Reply:
glTF spec for wrapping: https://github.com/KhronosGroup/glTF/tree/master/specification/2.0#samplerwraps

Reply:
Also ThreeJS doesn't support those? https://threejs.org/docs/index.html#api/en/constants/Textures